tools/installPluginVersion.php

```diff
@@ -68,11 +68,23 @@ function execute() {
 		if ($plugin && is_object($plugin)) {
 			PluginRegistry::register($matches[1], $plugin, $pluginPath);
 		}
+		$plugin = PluginRegistry::getPlugin($matches[1], $plugin->getName());
```

Comment:
Is this line necessary?

Reply:
For some strange reason without that line, the plugin object does not contain anythig, no pluginPath and no pluginCategory :-O